src/wallet/test/wallet_tests.cpp

```diff
@@ -69,32 +68,30 @@ BOOST_AUTO_TEST_CASE(coin_selection_tests)
     CoinSet setCoinsRet, setCoinsRet2;
     CAmount nValueRet;
 
-    LOCK(wallet.cs_wallet);
-
```

Comment:
Given that `add_coin` now uses `pwalletMain`, this lock should be altered rather than removed:
```
    LOCK(pwalletMain->cs_wallet);
```

Reply:
We don't actually need the lock because all the tests here are for the static `SelectCoinsMinConf` function.